Services/Administration/OJS.Services.Administration.Business/Implementations/SubmissionsForProcessingBusinessService.cs

```diff
@@ -71,5 +71,7 @@ public void EnqueuePendingSubmissions()
                 .GetAwaiter()
                 .GetResult();
         }
+
+        public void DeleteProcessedSubmissionsOlderThanADay() => this.submissionsForProcessingData.Clean();
```

Comment:
the `Clear` method does not clear submissions older than a day, but all that are processing=false and processed=true

Reply:
The issue description says the background job should be implemented as in old judge and that we don't need SFPs ''once the submission is marked for processed''.  My thought flow was that since it runs everyday and clears the processed submission from the day before - it is doing. Exactly logic should be discussed.